ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -251,6 +251,15 @@ def next_record(self) -> Iterable[Entity]:
 
                 if self.source_config.includeTables:
                     yield from self.fetch_tables(inspector, schema)
+
+                if self.source_config.schemaFilterPattern:
+                    if self.source_config.schemaFilterPattern.includes:
```

Comment:
Use the filter module here https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/utils/filters.py

Reply:
nvm, all this addition can be removed. It is done at line 246!